.docker/entrypoint/lib/context.rb

```diff
@@ -41,10 +43,18 @@ def arguments=(args)
         args.must_be_a! Arguments
         @arguments = args
       end
-
+      
       def verbose?
         @debug || arguments.log_level == :debug
       end
+      
+      def profile?
+        puts "Profile?"
```

Comment:
Codacy found an issue: [Prefer single-quoted strings when you don't need string interpolation or special symbols.](https://app.codacy.com/gh/SwedbankPay/jekyll-plantuml-docker/pullRequest?prid=6123667)